.devcontainer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:21-alpine3.18
+RUN apk add --no-cache \
+                git \
+                nano\
+                openssh
```

Comment:
Why do you add `openssh` ?

Reply:
To be able to use `ssh`keys to sign commits and login to github via ssh

https://github.com/pi-hole/web/blob/dc2cfdba28108b0e41f5b51155f48e41533c8d6b/.devcontainer/devcontainer.json#L20-L21

Reply:
👌